src/runtime/StaticStrings.cpp

```diff
@@ -26,92 +26,100 @@ namespace Escargot {
+    INIT_STATIC_STRING(stringCase, "case");
+    INIT_STATIC_STRING(stringVoid, "void");
+    INIT_STATIC_STRING(stringWith, "with");
+    INIT_STATIC_STRING(stringEnum, "enum");
+    INIT_STATIC_STRING(stringAwait, "await");
+    INIT_STATIC_STRING(stringWhile, "while");
+    INIT_STATIC_STRING(stringBreak, "break");
+    INIT_STATIC_STRING(stringThrow, "throw");
+    INIT_STATIC_STRING(stringConst, "const");
+    INIT_STATIC_STRING(stringClass, "class");
+    INIT_STATIC_STRING(stringSuper, "super");
+    INIT_STATIC_STRING(stringReturn, "return");
+    INIT_STATIC_STRING(stringTypeof, "typeof");
+    INIT_STATIC_STRING(stringSwitch, "switch");
+    INIT_STATIC_STRING(stringExport, "export");
+    INIT_STATIC_STRING(stringImport, "import");
+    INIT_STATIC_STRING(stringFinally, "finally");
+    INIT_STATIC_STRING(stringExtends, "extends");
+    INIT_STATIC_STRING(stringFunction, "function");
+    INIT_STATIC_STRING(stringContinue, "continue");
+    INIT_STATIC_STRING(stringDebugger, "debugger");
+    INIT_STATIC_STRING(stringInstanceof, "instanceof");
+    INIT_STATIC_STRING(stringImplements, "implements");
+    INIT_STATIC_STRING(stringInterface, "interface");
+    INIT_STATIC_STRING(stringPackage, "package");
+    INIT_STATIC_STRING(stringYield, "yield");
+    INIT_STATIC_STRING(stringLet, "let");
+    INIT_STATIC_STRING(stringError, "error");
+
+    INIT_STATIC_STRING(defaultRegExpString, "(?:)");
+
+    INIT_STATIC_STRING(get__proto__, "get __proto__");
+    INIT_STATIC_STRING(set__proto__, "set __proto__");
+    INIT_STATIC_STRING(getbyteLength, "get byteLength");
+    INIT_STATIC_STRING(getbyteOffset, "get byteOffset");
+    INIT_STATIC_STRING(getLength, "get length");
+    INIT_STATIC_STRING(getBuffer, "get buffer");
+    INIT_STATIC_STRING(getSize, "get size");
+    INIT_STATIC_STRING(getFlags, "get flags");
+    INIT_STATIC_STRING(getGlobal, "get global");
+    INIT_STATIC_STRING(getIgnoreCase, "get ignoreCase");
+    INIT_STATIC_STRING(getMultiline, "get multiline");
+    INIT_STATIC_STRING(getSource, "get source");
+    INIT_STATIC_STRING(getSticky, "get sticky");
+    INIT_STATIC_STRING(getUnicode, "get unicode");
+    INIT_STATIC_STRING(getSymbolSpecies, "get [Symbol.species]");
+    INIT_STATIC_STRING(getSymbolSearch, "get [Symbol.search]");
+    INIT_STATIC_STRING(getSymbolToStringTag, "get [Symbol.toStringTag]");
+    INIT_STATIC_STRING(symbolSplit, "[Symbol.split]");
+
+    INIT_STATIC_STRING($Ampersand, "$&");
+    INIT_STATIC_STRING($PlusSign, "$+");
+    INIT_STATIC_STRING($GraveAccent, "$`");
+    INIT_STATIC_STRING($Apostrophe, "$'");
+#undef INIT_STATIC_STRING
+
+#define INIT_STATIC_NUMBER(num) numbers[num].initStaticString(atomicStringMap, new ASCIIString(#num, strlen(#num)));
+    FOR_EACH_STATIC_NUMBER(INIT_STATIC_NUMBER)
+#undef INIT_STATIC_NUMBER
+
+    for (unsigned i = 0; i < ESCARGOT_ASCII_TABLE_MAX / 2; i++) {
+        const char c = (const char)i;
+        asciiTable[i].init(atomicStringMap, &c, 1);
```

Comment:
Why did you split the character initialization process into ASCII and Latin1 part? I'm just asking you because of my curiosity. This patch looks good to me:)

Reply:
It's all about categorization. ASCII strings are represents characters from 0 - 128 range and Latin1 strings from 0- 255 range. Therefore half of the single characters can be stored as ASCII strings.

Reply:
I see. Thank you for your kind explanation:)